backend/backend/settings.py

```diff
@@ -0,0 +1,4 @@
+from google.oauth2 import service_account
+
+client_file = 'sa-mock-ai-interviewer'
```

Comment:
No point defining this and the `.json` part seperately (on the next line)
Why not just in one constant together? e.g. CLIENT_FILE = 'sa-mock-ai-interviewer.json'